packages/apps-routing/src/contracts.ts

```diff
@@ -23,13 +23,14 @@ export default function create (t: TFunction): Route {
   return {
     Component,
     display: {
-      needsAccounts: true,
+      needsAccounts: false,
```

Comment:
OMG, it was that easy? 🤯 

There is tone of value in being able to call read-only methods w/o having to connecting an account.

Reply:
Yes, easy